config/feature_dependencies.gradle

```diff
@@ -2,10 +2,10 @@ apply plugin: 'kotlin-android'
 apply plugin: 'kotlin-kapt'
 
 android {
-    compileSdkVersion 30
+    compileSdkVersion 31
```

Comment:
When updating to Android 12, some changes must be done in the application as well.
One that I recall is that every Android component (Activity, BroadcastReceiver, ContentProvider and Services) must declare the `exported` tag on Manifest.

Please, take a look on [Android 12 behavior changes](https://developer.android.com/about/versions/12/behavior-changes-12) in order to avoid your app to crash in runtime.